src/Abp.AspNetCore/AspNetCore/AbpAspNetCoreModule.cs

```diff
@@ -25,6 +26,7 @@ public class AbpAspNetCoreModule : AbpModule
         public override void PreInitialize()
         {
             IocManager.Register<IAbpAspNetCoreConfiguration, AbpAspNetCoreConfiguration>();
+            IocManager.Register<IAbpActionResultWrapperFactory, AbpActionResultWrapperFactory>();
```

Comment:
Don't manually register it, but add ITransientDependency interface to AbpActionResultWrapperFactory. If you think that why IAbpAspNetCoreConfiguration is registered here, because it should be injectable in PreInitialize of other modules.